tests/core/conversion/converters/test_select.cpp

```diff
@@ -1093,6 +1093,32 @@ TEST(Converters, ATenIndexTensorIdxsNoneConvertsCorrectly) {
   ASSERT_TRUE(torch_tensorrt::tests::util::almostEqual(jit_results[0], trt_results[0], 2e-6));
 }
 
+TEST(Converters, ATenIndexTensorNoneIdx1ConvertsCorrectly) {
+  const auto graph = R"IR(
+      graph(%x.1 : Tensor,
+            %index0 : Tensor):
+        %5 : NoneType = prim::Constant()
+        %18 : Tensor?[] = prim::ListConstruct(%5, %index0)
+        %19 : Tensor = aten::index(%x.1, %18)
+        return (%19))IR";
```

Comment:
@mfeliz-cruise  Can you let me know the original pytorch code that generated this graph ?

Reply:
`input_tensor = input_tensor[:, [2, 1, 0], ...]`